### DISCUSSION TITLE
``vercel dev`` keeps running my dev|build scripts whereas they are not activated in the project

### DISCUSSION THREAD

**Discussion by lorenzofox3** [vercel/vercel#4637] [Help] [answered]
Hello, 

I have a serverless API (which we can call service A) built with typescript:

* the folder ``api`` has the handlers which are built and deployed by vercel CLI
* these handlers are basically glue used to expose the serveless functions based on the vercel conventions. The actual "logic" code is in a ``src`` folder. I have a ``dev`` and ``build`` scripts which compile this typescript source code and run the tests while I am working on this API (service A), with no particular interest for the ``api`` folder. So far so good.

This API is used by another service (B). While developing the service B I wish to run a local version of A. I thought an easy way would be to run ``vercel dev`` in A to get a dev server without the need to create custom dev server or container etc. (at the end just like a "local" deployment)

However when running the command, vercel CLI keeps running my "dev" and "build" which I thought I had discarded in my project's settings:

<img width="956" alt="Screenshot 2020-06-12 at 09 20 10" src="https://user-images.githubusercontent.com/2402022/84502021-daea4400-acb7-11ea-9943-d89424769a01.png">

This prevents my local A server to run. If I remove ``dev`` and ``build`` scripts from the package.json, everything works smoothly, vercel CLI builds and run a local dev server.

Any clue or advice ?

In a general manner, do you have an example of "real world" cases for no framework serverless API built with typescript? Most of the examples (typescript gatsby) are pretty "dumb" with no test, no dev environment and I find difficult to build upon them

vercel cli version: 19.0.1 on node v14

**Comment by styfle** [comment, marked as answer]
Thanks, this looks like a bug.

If you change the Development Command to something like `echo "nothing"` or some other no-op, does that solve your problem?

**Reply by lorenzofox3** [reply]
Thanks for your reply

If I add a dev command, it works just fine 

<img width="656" alt="Screenshot 2020-06-12 at 17 10 42" src="https://user-images.githubusercontent.com/2402022/84518001-51933b80-acd0-11ea-90ee-f9a8cb154920.png">

<img width="984" alt="Screenshot 2020-06-12 at 17 10 52" src="https://user-images.githubusercontent.com/2402022/84517917-36283080-acd0-11ea-982b-6215b7ef33ee.png">

Only if I leave it empty (which in my case is necessary) the problem occurs.

**Reply by styfle** [reply]
Thanks for confirming!

We are tracking this bug internally in ticket 3168 and will get this fixed!

**Reply by Igloczek** [reply]
for  me it throws errors now
```
Vercel CLI 23.1.2 dev (beta) — https://vercel.com/feedback
> Running Dev Command “echo "development"”
npm ERR! could not determine executable to run

npm ERR! A complete log of this run can be found in:
```
